src/frontend-samples/clashdetection-sample/ClashMarkers.ts

```diff
@@ -0,0 +1,230 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Is this significantly different from what's already in the marker pin sample?  If not, it would be better to share that code than to have a copy.

Reply:
I need to review this with you to figure out how to call clash specific visualization code on mouse button.

Reply:
Done - please review